modules/users/client/controllers/password.client.controller.js

```diff
@@ -55,13 +53,13 @@
     function onRequestPasswordResetSuccess(response) {
       // Show user success message and clear form
       vm.credentials = null;
-      vm.success = response.message;
+      Notification.success({ message: response.message, title: 'Password request successful!' });
```

Comment:
Sure. I understand. Just thought that the title wasn't very clear, or descriptive, for a user. Feel free to leave it as is.